apps/src/codebridge/FilePreview/InnerHTMLPreview.tsx

```diff
@@ -13,6 +13,7 @@ import {
   updateLinksToNonHtmlFiles,
 } from './htmlParsingHelpers';
 import PageNotFound from './PageNotFound';
+import PreviewEmptyState from './PreviewEmptyState';
```

Comment:
Do we need to implement this for `InnerHTMLPreview2` as well? https://github.com/code-dot-org/code-dot-org/pull/69422 - or will that be done in a follow-up?

Reply:
Yeah we'll have to handle it for the new preview too. It will be handled differently though. The 404 page should stay the same, and maybe in `HTMLPreview` instead of trying to show the preview if we have no sources we can show the empty state instead.

Reply:
We can do that separately though/wait for the new preview to be the default.